src/target/ch32f1.c

```diff
@@ -172,7 +172,7 @@ bool ch32f1_probe(target *t)
 	const uint32_t device_id = dbgmcu_idcode & 0x00000fffU;
 	const uint32_t revision_id = (dbgmcu_idcode & 0xffff0000U) >> 16;
 
-	DEBUG_WARN("DBGMCU_IDCODE %x, DEVID %x, REVID %x \n", dbgmcu_idcode, device_id, revision_id);
+	DEBUG_WARN("DBGMCU_IDCODE %lx, DEVID %lx, REVID %lx \n", dbgmcu_idcode, device_id, revision_id);
```

Comment:
These should use PRIx32 to fix the warnings across all platforms - these are still wrong on some platforms